model/host/db.go

```diff
@@ -78,6 +78,12 @@ var (
 	SpawnOptionsBuildIDKey       = bsonutil.MustHaveTag(SpawnOptions{}, "BuildID")
 	SpawnOptionsTimeoutKey       = bsonutil.MustHaveTag(SpawnOptions{}, "TimeoutTeardown")
 	SpawnOptionsSpawnedByTaskKey = bsonutil.MustHaveTag(SpawnOptions{}, "SpawnedByTask")
+	DistroIDKey                  = bsonutil.MustHaveTag(distro.Distro{}, "Id")
```

Comment:
This shouldn't be needed - use the variable in model/distro

Reply:
Removed